tests/Model/ValueObject/String/Behaviour/MockHasMaxLength.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\ValueObject\String\Behaviour;
+
+class MockHasMaxLength
+{
+    use HasMaxLength;
+
+    private bool $success;
+
+    public function __construct(string $value, int $maxLength)
+    {
+        $this->hasMaxLength($value, $maxLength);
+
+        $this->success = true;
+    }
+
+    public function isSuccess(): bool
```

Comment:
There is no need for this, the throw of the exception is what should be checked.